README.md

```diff
@@ -17,14 +17,13 @@ import './demo.css'
 
 const Demo = () => {
   const placementName = 'PublisherName_970x250_728x90_320x50'
-  const targeting = { key1: 'value1', key2: 'value2' }
-
+  const publisher = 'publisherName';
```

Comment:
for my sanity, can we stay consistent and remove the semi

Reply:
yup will change